### DISCUSSION TITLE
HTML Tutoriales

### DISCUSSION THREAD

**Discussion by 0Dimitry** [Launch-X-Latam/MisionFrontEnd#168] [General]
Para aquellos que esten interesados en complemetar lo que vimos y vamos a ver en HTML les recomiendo la sigueinte pagina:
https://www.w3schools.com/html/default.asp

**Comment by doguedogue** [comment]
Genial, gracias, también te recomiendo https://fernando-herrera.com/ 🚀🚀🚀

**Comment by doguedogue** [comment]
No me da comisión eh, solo he aprendido bastante con él hahaha tiene varios cursos gratis!
